### PR TITLE
INP doc updates.

### PR DESCRIPTION
**Pull request by malchata**
This adds the following suggestions to the INP doc:

- Better diagrams for illustrating interactions.
- Adds a vanilla JavaScript snippet for measuring INP.

### CODE REVIEW

File: src/site/content/en/metrics/inp/index.md [hunk 1]
```diff
@@ -252,6 +259,40 @@ See the [`onINP()`](https://github.com/GoogleChrome/web-vitals/tree/next#oninp)
 Gathering INP metrics in the field will only work on browsers that [fully support the Event Timing API](https://caniuse.com/mdn-api_performanceeventtiming), including its `interactionId` property.
 {% endAside %}
 
+You can also measure INP in browsers that support the Event Timing API without a library using vanilla JavaScript available in the expandable section below.
```
Comment (philipwalton) [inline src/site/content/en/metrics/inp/index.md:262]: This snippet doesn't actually measure INP. I think our message with this section/snippet should be something like:
- Measuring INP in JS is tricky, so we recommend the web-vitals lib for  most accurate results
- But to get a quick sense of what the longest interactions look like on your page, you can run this code (and modify it to suit your needs).

File: src/site/content/en/metrics/inp/index.md [hunk 2]
```diff
@@ -252,6 +259,40 @@ See the [`onINP()`](https://github.com/GoogleChrome/web-vitals/tree/next#oninp)
...
+
+{% Details %}
+
+{% DetailsSummary %}
+Snippet to measure INP in vanilla JavaScript
+{% endDetailsSummary %}
+
+```js
```
Comment (philipwalton) [inline src/site/content/en/metrics/inp/index.md:270]: Can you update this entire snippet with this version?

```js
let maxDuration = 0;

new PerformanceObserver(list => {
  for (const entry of list.getEntries()) {
    // Comment this out to show ALL event entry types (useful e.g. on Firefox).
    if (!entry.interactionId) continue;
    
    if (entry.duration > maxDuration) {
      // New longest Interaction to Next Paint (duration).
      maxDuration = entry.duration;
      console.log(`[INP] duration: ${entry.duration}, type: ${entry.name}`, entry);
    } else {
      // Not the longest Interaction, but uncomment the next line if you still want to see it.
      // console.log(`[Interaction] duration: ${entry.duration}, type: ${entry.name}`, entry);
    } 
  }
}).observe({
  type: 'event',
  durationThreshold: 16, // Minimum supported by the spec.
  buffered: true
});
```

**Review by philipwalton — CHANGES_REQUESTED** [review]
Did you also want to comment out mentions of the tools that haven't launched yet?

**Comment by malchata** [comment]
@philipwalton, I just added changes for you to re-review.

**Review by philipwalton — APPROVED** [review]

**Pull request merged**